armi/materials/custom.py

```diff
@@ -40,7 +40,7 @@ def __init__(self):
         by a constant user-input density.
         """
         Material.__init__(self)
-        self.p.density = 1.0
+        self.customDensity = 1.0
```

Comment:
kind of a minor point but if we're renaming things like this it might be wise to make them supernames like `customDensityInGPerCC`. As it turns out, putting units in names is one honkin good idea, especially for super generic terms like 'density'